src/components/Canvas/service/image_management_service.js

```diff
@@ -1,19 +1,23 @@
 const getFactor = (height, width) => {
   switch (true) {
+    case width >= 1600 && height >= 1550:
+      return {sizeFactor: 9, moveWidthFactor: 2.8, moveTopHeightFactor: 1.2, moveBottomHeightFactor: 1.7, moveCentralHeightFactor: 15};
```

Comment:
can you add a comment on top what means `sizeFactor ` `moveWidthFactor` `moveTopHeightFactor` `moveBottomHeightFactor` `moveCentralHeightFactor`

Reply:
Done